src/python/WMComponent/JobSubmitter/JobSubmitterPoller.py

```diff
@@ -254,10 +256,13 @@ def refreshCache(self):
 
         logging.info("Querying WMBS for jobs to be submitted...")
         logging.info("cachedJobIDs : %s" % len(self.cachedJobIDs))
-        if len(self.cachedJobIDs) < self.cacheRefreshSize:
+        if self.cacheRefreshSize == -1 or len(self.cachedJobIDs) < self.cacheRefreshSize or \
```

Comment:
In this case we just need to set 'cacheRefreshSize' to 0. It would refresh all the time :-)